content/applications/general/companies/digest_emails.rst

```diff
@@ -1,83 +1,209 @@
+- :guilabel:`Next Send Date`: the date on which the digest email will be sent again.
+- :guilabel:`KPIs` tab: check/uncheck each calculated :abbr:`KPIs (key performance indicators)` that
+  appears in Digest Emails.
+- :guilabel:`Recipients` tab: add/remove users who receive the Digest Emails.
 
-From there, give your Digest Email a title, specify periodicity, and choose your desired KPIs and
-recipients fields as needed.
+From there, give the digest email a :guilabel:`Digest Name`, specify :guilabel:`Periodicity`,
+choose the desired :abbr:`KPIs (key performance indicators)`, and add :guilabel:`Recipients` as
+needed.
 
-After you click **Save**, your new custom Digest Email is available as a selection in the **General
-Settings** dropdown menu.
+After clicking :guilabel:`Save`, the new custom digest email is available as a selection in the
+:guilabel:`General Settings` drop-down menu.
 
 .. _digest-emails/custom-kpi:
 
 Custom KPIs with Studio
------------------------
+=======================
+
+The :abbr:`KPIs (key performance indicators)` can be customized using Odoo *Studio*.
+
+.. warning::
+   Additional costs to the database subscription will be incurred should *Studio* need to be
+   installed.
+
+For either *Your Odoo Periodic Digest* or a custom digest email, :abbr:`KPIs (key performance
+indicators)` can be added by using Odoo *Studio*.
+
+To begin, click the :guilabel:`🛠️ (tools)` icon in the top right of the screen. This is the link to
+the *Studio* application.
 
-For either *Your Odoo Periodic Digest* or your own custom Digest Email, you can add your own KPIs
-by using Odoo Studio.
+In order to create additional fields, create two fields on the digest object:
 
-To begin, click the **Toggle Studio** icon or click the **Recipients** tab and then the ellipses
-:menuselection:`… icon --> Add Custom Field` to edit the template or add additional fields.
+#. Create a boolean field called `kpi_myfield` and display it in the :guilabel:`KPIs` tab.
+#. Create a computed field called `kpi_myfield_value` that computes the customized :abbr:`KPI (key
+   performance indicator)`.
+#. Select the :abbr:`KPIs (key performance indicators)` in the :guilabel:`KPIs` tab.
 
-In order to create additional fields, you must create two fields on the digest object:
+.. tip::
+   Here is the `source code
+   <https://github.com/odoo/odoo/blob/aa09df25845f5497124185013bdb5706fcb2b082/addons/digest/models/digest.py>`_
+   for the digest.py file, which will guide the programmer in the coding of the computed field.
 
-#. create a boolean field called `kpi_myfield` and display it in the KPIs tab
-#. create a computed field called `kpi_myfield_value` that computes your customized KPI
-#. select your KPI(s) in the KPIs tab.
+.. seealso::
+   An alternative is to click the :guilabel:`Recipients` tab and then the :guilabel:`...` (ellipses)
+   to edit this view. Either click :guilabel:`EDIT LIST VIEW` or :guilabel:`EDIT FORM VIEW` to
+   modify this tab.
```

Comment:
I'm not seeing the "..." in the Recipients tab of a digest email template form. Can you please double-check the accuracy of this during your revisions? Thanks! 👍

Reply:
Good catch!!